### PR TITLE
Issue #15218: formatted input files for section 7.1.2 paragraphs to follow google style guide rules

### PR DESCRIPTION
**Pull request by Zopsss**
#15218

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraph.java [hunk 1]
```diff
@@ -6,91 +6,97 @@
...
 //  '\<p\> tag should be placed immediately before the first word'
 //  '\<p\> tag should be preceded with an empty line.'
 
 class InputIncorrectJavadocParagraph {
 
+  // 2 violations 4 lines below:
+  //  '\<p\> tag should be placed immediately before the first word'
+  //  '\<p\> tag should be preceded with an empty line.'
+  /**
+   * Some Javadoc.<p>
```
Comment (rdiachenko) [inline src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraph.java:21]: I'm curious: was the `P` changed to lowercase by google formatter?

**Review by rdiachenko — APPROVED** [review]
lgtm

File: src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraph.java (hunk 1 above)
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraph.java:21]: Yes. Formatter changed it to lowercase and also fixed its position but we had to show the violation so I reverted back its position but did not converted it back to the uppercase

**Review by romani — APPROVED** [review]
ok to merge

**Pull request merged**